test/run-tests

```diff
@@ -216,9 +221,18 @@ def run(*argv, env=None, check=True, cwd=None):
         runenv.update(env)
 
     print("+ " + cwdrepr + envrepr + " ".join(shlex.quote(a) for a in argv))
+    stdout_val = sys.stdout
+    if return_stdout:
+        stdout_val = subprocess.PIPE
 
     return subprocess.run(
-        argv, env=runenv, check=check, stdout=sys.stdout, stderr=sys.stderr, cwd=cwd
+        argv,
+        env=runenv,
+        check=check,
+        stdout=stdout_val,
+        stderr=sys.stderr,
+        cwd=cwd,
+        text=text,
```

Comment:
this change was made because python `black` did not like the old formatting . . .